src/components/Favorites.tsx

```diff
@@ -1,9 +1,29 @@
-import MoviesList from './MoviesList';
+import { useEffect, useState } from 'react';
+import api from 'src/service/api';
+import RenderResults from './RenderResults';
 
 const Favorites = () => {
+  const [data, setData] = useState([{
+    poster_path: ' ',
+    original_title: ' ',
+    title: ' ',
+    vote_average: 0,
+    vote_count: ' ',
+    overview: ' ',
+    backdrop_path: ' ',
+    id: 0,
```

Comment:
Why do we need to have this dummy object? Why not []?

Reply:
If we add empty array then ts will be to argue with me))))